src/Effects.js

```diff
@@ -682,6 +670,85 @@ module.exports = class Effects {
       }
     };
 
+    this.hearts = {
+      heart: [],
+      size: 50,
+      init: function () {
+        for (let i = 0; i < 20; i++) {
+          this.heart.push({
+            x: randomNumber(10, 390),
+            y: -50,
+            rot: randomNumber(0, 359),
+            speed: p5.random(1,5),
```

Comment:
I missed this somehow - why are we using `randomNumber` in some cases and `p5.random` in others?

Reply:
as discussed, probably just carry over from re-using other effects that chose different styles. will look to standardize across the file as part of a clean up PR later.